geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSetCommandsFunctionExecutor.java

```diff
@@ -62,6 +62,13 @@ public long zcount(RedisKey key, SortedSetRangeOptions rangeOptions) {
         () -> getRedisSortedSet(key, true).zrange(min, max, withScores));
   }
 
+  @Override
+  public List<byte[]> zrangebyscore(RedisKey key, SortedSetRangeOptions rangeOptions,
+      boolean withScores) {
+    return stripedExecute(key,
+        () -> getRedisSortedSet(key, true).zrangebyscore(rangeOptions, withScores));
```

Comment:
I noticed you are calling getRedisSortedSet with updatesStats set to true. This seems reasonable. But I also noticed that a bunch of other methods on this class (zadd, zincrby, zrange, zrem) call it with false. Do you know when to set it to true vs false?

Reply:
I believe that Redis' approach is to update the stats for commands that don't update data. I would expect that `zrange` _should_ update these stats.

Reply:
I see ZRANGE calling it with true. In general, operations that read data update stats, operations that change data do not. So zadd, zrem, zincrby change data and therefore don't update stats.